pkg/manifest/manifest.go

```diff
@@ -383,6 +403,14 @@ func (manifest *manifest) PlatformConfigFileName() (string, error) {
 	return string(manifest.Id) + "_" + sha + ".platform.yaml", nil
 }
 
+func (manifest *manifest) ResourceLimitsConfigFileName() (string, error) {
+	sha, err := manifest.SHA()
+	if err != nil {
+		return "", err
+	}
+	return string(manifest.Id) + "_" + sha + ".resource.yaml", nil
```

Comment:
nit: `manifest.Id.String()`. I like those functions because if we ever change the way the type works then the compiler will warn us, whereas casting to a string might not